Userland/Libraries/LibWeb/XHR/XMLHttpRequest.cpp

```diff
@@ -101,13 +102,20 @@ void XMLHttpRequest::visit_edges(Cell::Visitor& visitor)
 // https://xhr.spec.whatwg.org/#concept-event-fire-progress
 static void fire_progress_event(XMLHttpRequestEventTarget& target, FlyString const& event_name, u64 transmitted, u64 length)
 {
-    // To fire a progress event named e at target, given transmitted and length, means to fire an event named e at target, using ProgressEvent,
-    // with the loaded attribute initialized to transmitted, and if length is not 0, with the lengthComputable attribute initialized to true
-    // and the total attribute initialized to length.
+    // To fire a progress event named e at target, given transmitted and length,
+    // means to fire an event named e at target, using ProgressEvent, with
```

Comment:
This trailing "with" feels like it belongs better the start of the comment on 109.

Reply:
sure